src/main/java/frc/robot/MotorController.java

```diff
@@ -35,6 +35,7 @@ public MotorController(String name, int deviceID, int smartCurrentLimit) {
         this(name, deviceID); //calls the constructor above
         //Current limiting is required to prevent brown outs
         mSparkMax.setSmartCurrentLimit(smartCurrentLimit);
+        mSparkMax.setOpenLoopRampRate(.1);
```

Comment:
Did this have a noticeable effect on driving? 
API doc says "Time in seconds to go from 0 to full throttle."  
100 ms seems faster than the motors are capable of

Reply:
It helps make driving more smooth. The ramp rate is for the voltage command for the motor controller, so it takes 100 ms to send full voltage to the motor.